File: front/src/app/models/reserva/reserva.ts
import { Empresa } from '../empresa/empresa';
import { Usuario } from '../usuario/usuario';

export interface Reserva {
  id: string;
  usuarioId: string;
  usuario: Usuario[];
  empresaId: string;
  empresa: Empresa[];
  concluida: boolean;
  dataHoraReserva: Date;
}

File: SemTumultoApi/Models/Reservas/Reserva.cs
using System;
using SemTumultoApi.Models.Empresas;
using SemTumultoApi.Models.Usuarios;

namespace SemTumultoApi.Models.Reservas
{
    public class Reserva
    {
        public Guid Id { get; set; }
        public Guid? UsuarioId { get; set; }
        public Usuario Usuario { get; set; }
        public Guid? EmpresaId { get; set; }
        public Empresa Empresa { get; set; }
        public bool Concluida { get; set; } = false;
        public DateTime DataHoraReserva { get; set; }

    }
}
